crates/rspack/tests/tree-shaking/reexport-all-as-multi-level-nested/expected/main.js

```diff
@@ -0,0 +1,58 @@
+(self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
+"./package/autogen/a.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  'aa': function() { return _aa__WEBPACK_IMPORTED_MODULE_0_; }
+});
+__webpack_require__.d(__webpack_exports__, {
+  'a': function() { return a; }
+});
+/* harmony import */var _aa__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./aa */"./package/autogen/aa.js");
+
+ function a() {}
+ function dddd() {}
+
+},
+"./package/autogen/aa.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  'aa': function() { return aa; }
```

Comment:
https://github.com/web-infra-dev/rspack/pull/3895/files#diff-b9f6a6ade6a2bbb6665d164b6cfdd1e1b919a7f3fd2181815f0391d5b8a3f092R3, only `aa` is used.